ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -392,3 +392,47 @@ def verify_failover_relocate_status_ui(
         log.info("Close button found")
         acm_obj.do_click_by_xpath("//*[text()='Close']")
         log.info("Data policy modal page closed")
+
+
+def delete_application_ui(acm_obj, workload_to_delete=None, timeout=60):
+    """
+    Function to delete specified workloads on ACM UI
+
+    Args:
+        acm_obj (AcmAddClusters): ACM Page Navigator Class
+        workload_to_delete (str): Specify the workload to delete, otherwise none will be deleted
+        timeout (int): timeout to wait for certain elements to be found on the ACM UI
+
+    """
+    if workload_to_delete:
```

Comment:
workload_to_delete isn't this required value, why setting it to None?

Reply:
we can pass it in the testcase

Reply:
done